main.go

```diff
@@ -37,33 +37,50 @@ var (
 	tab       = flag.Int("tab", 8, "tab width for column calculations")
 	overwrite = flag.Bool("w", false, "overwrite modified files")
 	fast      = flag.Bool("fast", false, "skip running goimports")
+	printDiff = flag.Bool("diff", true, "print diffs")
 	ignore    = flag.String("ignore", "", "regex matching files to skip")
 )
 
 func main() {
 	flag.Parse()
 	if len(flag.Args()) != 1 {
```

Comment:
you mean `if len(flag.Args()) == 0` here and then below error out if it's `>1`.

Reply:
Done